[2016-08-11 09:18:47] local.ERROR: exception 'BadMethodCallException' with message 'Method [surveyForm] does not exist.' in E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Controller.php:268
Stack trace:
#0 [internal function]: Illuminate\Routing\Controller->__call('surveyForm', Array)
#1 [internal function]: SkillsMapController->surveyForm()
#2 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(231): call_user_func_array(Array, Array)
#3 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(93): Illuminate\Routing\Controller->callAction('surveyForm', Array)
#4 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(62): Illuminate\Routing\ControllerDispatcher->call(Object(SkillsMapController), Object(Illuminate\Routing\Route), 'surveyForm')
#5 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(967): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'SkillsMapContro...', 'surveyForm')
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Route.php(109): call_user_func_array(Object(Closure), Array)
#8 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1033): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1001): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#10 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(775): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(745): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Session\Middleware.php(72): Illuminate\Foundation\Application->handle(Object(Illuminate\Http\Request), 1, true)
#13 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Cookie\Queue.php(47): Illuminate\Session\Middleware->handle(Object(Illuminate\Http\Request), 1, true)
#14 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Queue->handle(Object(Illuminate\Http\Request), 1, true)
#15 E:\xampp\htdocs\connect\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#16 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#17 E:\xampp\htdocs\connect\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2016-08-11 09:19:52] local.ERROR: exception 'BadMethodCallException' with message 'Method [surveyForm] does not exist.' in E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Controller.php:268
Stack trace:
#0 [internal function]: Illuminate\Routing\Controller->__call('surveyForm', Array)
#1 [internal function]: SkillsMapController->surveyForm()
#2 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(231): call_user_func_array(Array, Array)
#3 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(93): Illuminate\Routing\Controller->callAction('surveyForm', Array)
#4 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(62): Illuminate\Routing\ControllerDispatcher->call(Object(SkillsMapController), Object(Illuminate\Routing\Route), 'surveyForm')
#5 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(967): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'SkillsMapContro...', 'surveyForm')
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Route.php(109): call_user_func_array(Object(Closure), Array)
#8 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1033): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1001): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#10 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(775): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(745): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Session\Middleware.php(72): Illuminate\Foundation\Application->handle(Object(Illuminate\Http\Request), 1, true)
#13 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Cookie\Queue.php(47): Illuminate\Session\Middleware->handle(Object(Illuminate\Http\Request), 1, true)
#14 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Queue->handle(Object(Illuminate\Http\Request), 1, true)
#15 E:\xampp\htdocs\connect\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#16 E:\xampp\htdocs\connect\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#17 E:\xampp\htdocs\connect\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2016-08-11 09:59:18] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')', expecting ',' or ';'' in E:\xampp\htdocs\connect\app\storage\views\66dcab4cf48f3c66f42010fa91049664:50
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
